src/tests/ftest/util/scrubber_test_base.py

```diff
@@ -131,3 +132,40 @@ def run_ior_and_check_scruber_status(self, pool, cont, fail_on_warning=True):
         # Just make sure scrubber is working here.
         status = self.verify_scrubber_metrics_value(self.initial_metrics, self.final_metrics)
         return status
+
+    def run_mdtest_and_check_scruber_status(self, pool, cont, mdtest_params):
+        """Run mdtest and get scrubber metrics
+
+        Args:
+            pool (object): Pool object
+            cont (object): Container object within the pool.
+            mdtest_params(list): List comprising of different set of mdtest parameters.
+
+        Returns:
+            status(bool) : True (Scrubber working), False(Scrubber not working)
+        """
+        status = False
+        self.initial_metrics = self.scrubber.get_csum_total_metrics()
+        self.pool = pool
+        self.container = cont
+        # Print the pool properties
+        result = self.dmg_cmd.pool_get_prop(self.pool.uuid, "scrub")
+        self.log.info("Pool Properties")
+        self.log.info("===============")
+        self.log.info(result)
```

Comment:
It's not necessary to print the pool properties because it's already in the log. E.g.
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13199/9/artifact/Functional%20Hardware%20Medium/scrubber/check_csum_metrics_mdtest.py/job.log
```
2023-11-22 03:15:14,482 process          L0604 INFO | Running '/usr/bin/dmg -o /etc/daos/daos_control.yml -d -j pool get-prop 4E13A9D3-7899-41C2-99B4-95502F72B8BF scrub'
2023-11-22 03:15:14,590 process          L0416 DEBUG| [stderr] DEBUG 03:15:14.590328 main.go:209: debug output enabled
2023-11-22 03:15:14,590 process          L0416 DEBUG| [stderr] DEBUG 03:15:14.590754 main.go:241: control config loaded from /etc/daos/daos_control.yml
2023-11-22 03:15:14,591 process          L0416 DEBUG| [stderr] DEBUG 03:15:14.591575 pool.go:911: pool get-prop request: *mgmt.PoolGetPropReq (sys:"daos_server-2.5.100" id:"4e13a9d3-7899-41c2-99b4-95502f72b8bf" properties:{number:16})
2023-11-22 03:15:14,592 process          L0416 DEBUG| [stderr] DEBUG 03:15:14.592112 rpc.go:278: request hosts: [wolf-142:10001 wolf-143:10001]
2023-11-22 03:15:14,855 process          L0416 DEBUG| [stderr] DEBUG 03:15:14.855253 response.go:168: wolf-142:10001: *mgmt.PoolGetPropResp status:DER_SUCCESS(0): Success (properties:{number:16 numval:2})
2023-11-22 03:15:14,855 process          L0416 DEBUG| [stdout] {
2023-11-22 03:15:14,855 process          L0416 DEBUG| [stdout]   "response": [
2023-11-22 03:15:14,855 process          L0416 DEBUG| [stdout]     {
2023-11-22 03:15:14,855 process          L0416 DEBUG| [stdout]       "name": "scrub",
2023-11-22 03:15:14,855 process          L0416 DEBUG| [stdout]       "description": "Checksum scrubbing mode",
2023-11-22 03:15:14,856 process          L0416 DEBUG| [stdout]       "value": "timed"
2023-11-22 03:15:14,856 process          L0416 DEBUG| [stdout]     }
2023-11-22 03:15:14,856 process          L0416 DEBUG| [stdout]   ],
2023-11-22 03:15:14,856 process          L0416 DEBUG| [stdout]   "error": null,
2023-11-22 03:15:14,856 process          L0416 DEBUG| [stdout]   "status": 0
2023-11-22 03:15:14,856 process          L0416 DEBUG| [stdout] }
2023-11-22 03:15:15,857 process          L0686 INFO | Command '/usr/bin/dmg -o /etc/daos/daos_control.yml -d -j pool get-prop 4E13A9D3-7899-41C2-99B4-95502F72B8BF scrub' finished with 0 after 1.2891182899475098s
2023-11-22 03:15:15,858 scrubber_test_ba L0153 INFO | Pool Properties
2023-11-22 03:15:15,858 scrubber_test_ba L0154 INFO | ===============
2023-11-22 03:15:15,858 scrubber_test_ba L0155 INFO | {'response': [{'name': 'scrub', 'description': 'Checksum scrubbing mode', 'value': 'timed'}], 'error': None, 'status': 0}
```
Also, there is a shorthand function for this
```suggestion
        # Print the pool properties
        self.log.info("Pool Properties")
        pool.get_prop("scrub")
```

Reply:
Removing it since its already printed.